ast/doc.go

```diff
@@ -2,5 +2,7 @@
 // Use of this source code is governed by an Apache2
 // license that can be found in the LICENSE file.
 
-// Deprecated: Use [github.com/open-policy-agent/opa/v1/ast] instead.
+// Deprecated: This package is intended for older projects transitioning from OPA v0.x and will remain for the lifetime of OPA v1.x, but its use is not recommended.
```

Comment:
Hmm. I feel like "remain" need one more word; but I could be wrong. "and will remain available"? 🤔